### ISSUE TITLE
"go install @latest" not working

### ISSUE THREAD

**Issue by barakplasma** [multiprocessio/dsq#61]
I tried installing latest (c6daddfd43b61c1a932404d88dd6e68035b0dd01) with
`go install github.com/multiprocessio/dsq@latest`
but it failed with 
```
go: downloading github.com/multiprocessio/dsq v0.0.0-20220515195645-c6daddfd43b6
go: github.com/multiprocessio/dsq@latest (in github.com/multiprocessio/dsq@v0.0.0-20220515195645-c6daddfd43b6):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```
however, this older version's install command worked
`go install github.com/multiprocessio/dsq@4ca4a314a0380c55ab9fc12bb03bc2c998d5580c`

I think the issue is related to this line https://github.com/multiprocessio/dsq/commit/4140b05c4e9fd6efbcedf23e58d6025385dbc42b#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R5

**Comment by eatonphil** [comment]
Hey that's unfortunate. Yes I'm using a different version of a 3rd party library in a few places while I wait for a PR in the 3rd party library to merge (if it ever merges).

Are you unable to use one of the binaries on the release page?

**Comment by barakplasma** [comment]
[latest release binary](https://github.com/multiprocessio/dsq/releases/tag/0.18.0) for linux works for me :+1: just not this command https://github.com/multiprocessio/dsq/blob/5b26feb6c4766a54c78ef49e3fade51d02d17f81/README.md?plain=1#L48-L55

**Comment by eatonphil** [comment]
Good point. I'll change that step. If you clone the repo and install with go build it should work.

**Comment by eatonphil** [comment]
Closed in https://github.com/multiprocessio/dsq/commit/72bf6bb283b139b49b3711f348f89a67bb4eaf67

**Comment by eatonphil** [comment]
You should be able to `go install` again now by the way.

**Comment by barakplasma** [comment]
confirmed that go install is working again. Thanks @eatonphil !
